pytest/tests/loadtest/contract/Cargo.toml

```diff
@@ -4,9 +4,6 @@ version = "0.1.0"
 authors = ["Near Inc <hello@near.org>"]
 edition = "2018"
 
-[lints]
-workspace = true
-
```

Comment:
I'm not familiar with lints, what does this do and why did you delete it?

Reply:
Doesn't compile otherwise